solr/core/src/java/org/apache/solr/pkg/PackageLoader.java

```diff
@@ -52,10 +56,14 @@
 public class PackageLoader implements Closeable {
   private static final Logger log = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
   public static final String LATEST = "$LATEST";
-
+  public static final String LOCAL_PACKAGES_DIR = System.getProperty("solr.packages.local.dir");
+  public static final String LOCAL_PACKAGES_WHITELIST = System.getProperty("solr.packages.local.whitelist", "");
```

Comment:
instead of `whitelist`, how about `names`, 'use' or `enable`. We are steering away from `whitelist`/`blacklist` now, and it doesn't work as well as other options.